src/components/payments/Payments.tsx

```diff
@@ -0,0 +1,68 @@
+import { useState } from 'react';
+
+import { ActionButton } from '../actionButton/ActionButton';
+import { Notification } from '@/entities/Notification';
+
+interface IPaymentsProps {
+  sendPayment: (senderPrivateKey: string, receiverPublicKey: string, amount: string) => Promise<boolean | string>;
+}
+
+export function Payments({ sendPayment }: IPaymentsProps): React.ReactElement {
+  const [amount, setAmount] = useState('');
+  const [senderPrivateKey, setSenderPrivateKey] = useState('');
+  const [receiverPublicKey, setReceiverPublicKey] = useState('');
+  const [paymentNotification, setPaymentNotification] = useState<Notification>();
+
+  async function handlePayment() {
+    setPaymentNotification({ isSuccess: true, message: 'Loading...' });
+    const result = await sendPayment(senderPrivateKey, receiverPublicKey, amount);
+
+    if (result === true) {
+      setPaymentNotification({ isSuccess: true, message: 'Success!' });
+    } else {
+      setPaymentNotification({ isSuccess: false, message: result });
+    }
```

Comment:
If there's a `result` it would return true. You don't have to compare it with a boolean.
```suggestion
    if (result) {
      setPaymentNotification({ isSuccess: true, message: 'Success!' });
    } else {
      setPaymentNotification({ isSuccess: false, message: result });
    }
```